hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsCustomEncryption.java

```diff
@@ -95,53 +110,58 @@ public class ITestAbfsCustomEncryption extends AbstractAbfsIntegrationTest {
   @Parameterized.Parameter(6)
   public boolean isCpkResponseHdrExpected;
 
+  @Parameterized.Parameter(7)
+  public Boolean isCpkResponseKeyExpected = false;
```

Comment:
if response is going to have cpk related values. Will be used in case of listsStatus. Ref: [learn.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/list#pathlist](https://learn.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/list#pathlist)